app/actions/local/channel.ts

```diff
@@ -189,7 +189,16 @@ export const markChannelAsViewed = async (serverUrl: string, channelId: string,
     }
 };
 
-export const markChannelAsUnread = async (serverUrl: string, channelId: string, messageCount: number, mentionsCount: number, manuallyUnread: boolean, lastViewed: number, prepareRecordsOnly = false) => {
+export const markChannelAsUnread = async (
+    serverUrl: string,
+    channelId: string,
+    messageCount: number,
+    mentionsCount: number,
+    manuallyUnread: boolean,
+    lastViewed: number,
+    isUnread: boolean | undefined,
```

Comment:
this can be inferred as if it is being marked as unread this is obviously going to always be **true**